src/components/editor/faultTree/Editor.tsx

```diff
@@ -186,6 +212,7 @@ const Editor = ({ setAppBarName }: DashboardTitleProps) => {
         onNodeMove={handleMoveEvent}
         setHighlightedElement={setHighlightedElementView}
         refreshTree={refreshTree}
+        faultEventScenarios={showPath ? faultEventScenarios : []}
```

Comment:
IMO `faultEventScenarios={showPath ? faultEventScenarios : []}` should be `faultEventScenarios={faultEventScenarios}` and the component should have another property `<EditorCanvas showPath={showPath}/>`

Reply:
Changed